### DISCUSSION TITLE
Journey to getting ADFS-OpenId Connect login working

### DISCUSSION THREAD

**Discussion by Sebazzz** [datalust/seq-tickets#2007] [General]
Just my quick notes here of my steps I needed to get ADFS login operational:

- Authority is `https://adfs.yourdomain.org/adfs/` (the `/adfs/` part being significant)
- The experimental feature flag of #1801 should be added to <data dir>\Seq.json:
```json
  "features": {
    "enabled": ["OidcAvoidUserInfoEndpoint"]
  },
```
- The `allatclaims` scope should be also be added to the requested claims. This is necessary, because although ADFS does return the claims in the access token, it does not in the `id_token` if you don't have this enabled.
- Add a custom claim rule to pass the `email_verified` claim: 
```
 => issue(Type = "email_verified", Value = "true");
```

**Comment by nblumhardt** [comment]
Fantastic - thank you, @Sebazzz! 😎
